src/core/directives/binding/binding_parser.ts

```diff
@@ -194,7 +194,7 @@ export function _setupOutputs(
  * @private
  * @deprecated
  */
-export function _parseBindings({ inputs=[], outputs=[], attrs=[] }): ParsedBindings {
+export function _parseBindings({ inputs=[], outputs=[], attrs=[] }: { inputs: string[], outputs: string[], attrs: string[] }): ParsedBindings {
```

Comment:
Yes, because `undefined` is now its own type - so `undefined[]` (cause by the parameter default value) is not assignable to `string[]`